recipe/meta.yaml

```
@@ -22,25 +26,24 @@ package:
 
 source:
   # Using github as source for mlflow-skinny which does not exist on PyPI
+  # Note: We use the default pyproject.toml (not pyproject.release.toml) because
+  # pyproject.release.toml declares mlflow as depending on mlflow-skinny and
+  # mlflow-tracing as pip packages, which would cause pip check to fail since
+  # those packages aren't installed in our build. Converting to a multi-output
+  # recipe would be needed to use pyproject.release.toml correctly.
   url: https://github.com/mlflow/mlflow/archive/refs/tags/v{{ version }}.zip
-  sha256: 910ad1e2fce93b412063637d465e9291c99678a3bbccbf2b4f7736887321f44b
+  sha256: 9cfa071e7a75ff4a0d47a3ee4bfc948a6dfa019edc2863325b71d2362f61b760
 
 build:
   number: 0
-  skip: true  # [py<39 or s390x]
+  skip: true  # [py<310 or s390x]
```

```suggestion
  skip: true  # [py<310]
```
You can drop s390x as we don't support it anymore.